imageio/plugins/ffmpeg.py

```diff
@@ -512,7 +514,8 @@ def _load_infos(self):
             if match is not None:
                 hms = map(float, line[match.start()+1:match.end()].split(':'))
                 self._meta['duration'] = duration = cvsecs(*hms)
-                self._meta['nframes'] = int(round(duration*fps))
+                if fps:
+                    self._meta['nframes'] = int(round(duration*fps))
```

Comment:
@kuchi This line here seems a bit fragile. I checked with the video file from #168, for which `duration*fps == 167.1`. So its not that we need to drop the `round()` call. But can we like verify this by seeking to the end or something? Maybe not in this PR, but do you think its possible?